.github/workflows/test.yml

```diff
@@ -0,0 +1,170 @@
+# yamllint disable rule:line-length
+name: Build and test
+
+on: # yamllint disable-line rule:truthy
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - main
+      - dev
+      - release*
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - master
+      - main
+      - dev
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  librocksdb:
+    name: Build librocksdb-${{ matrix.rocksdb-version }}
+    runs-on: ubuntu-22.04
```

Comment:
Is there any reason you chose Ubuntu 22.04 here, instead of ubuntu-latest like in other jobs?

Reply:
Yes, for some reason the compiler on `ubuntu-latest` gives too many warnings and will fail when compiling rocksdb.